src/config.json

```
@@ -723,6 +723,7 @@
     "askzeta.com"
   ],
   "blacklist": [
+    "metamạsk.io",
```

Should be in punycode

```suggestion
    "xn--metamsk-en4c.io",
```